stash/src/controller/tokenNetworkPortfolioController.ts

```diff
@@ -14,8 +14,10 @@ export const tokenNetworkPortfolio = async (req: Request, res: Response) => {
     )
     const portfolioBalance = accountBalances.map(
       async ([storageKey, value]) => {
-        const freeTokens = new BN(value.free.toString())
-        const frozenTokens = new BN(value.frozen.toString())
+        const free = JSON.parse(JSON.stringify(value)).free.toString()
```

Comment:
With upgrading polkadot/api version for some values instead of real value we started getting Codec and changes like the one seen here and toHuman() in other files are to solve that.

Reply:
looks like this is a types issue :(

Reply:
yes introduced with polkadot api version that we now use :)